### PR TITLE
[Oversite]: remove old example

### PR DESCRIPTION
**Pull request by zo-el**
This came up because of this issue #87

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,137 +21,149 @@ Take a look at the sample below, or skip to the [Conceptual Overview](#conceptua
...
-    [testDna] // contains 1 dna, the "test" dna
-  ]
-  // agent 1
-  [
+    [testDna], // contains 1 dna, the "test" dna
+  ][
+    // agent 1
     // happ 0
-    [dnaBlog], // contains 1 dna, the blog dna
+    ([dnaBlog], // contains 1 dna, the blog dna
```
Comment (Connoropolous) [inline README.md:50]: Some round brackets made it in here but I don't think they belong?

**Review by Connoropolous — COMMENTED** [review]
It's hard to tell the relevant changes with the linting, can you summarize or point to things @zo-el ?

**Comment by zo-el** [comment]
> It's hard to tell the relevant changes with the linting, can you summarize or point to things @zo-el ?

@Connoropolous - Sorry about the linting, this was just meant to remove the old example folder, since it caused some confusion with a community member. 
That is the only thing I have done here, apart from the README.md linting

**Comment by maackle** [comment]
Unless the README linting was done with a linter that the rest of the codebase adheres to, I think it should be removed. Also there is a legit concern by Connor about parentheses being added during linting which I think needs to be fixed.

**Comment by jost-s** [comment]
Old example was incidentally deleted along with #101
